akka-projection-core/src/test/scala/akka/projection/internal/ProjectionBehaviorSpec.scala

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection.internal
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.concurrent.Promise
+
+import akka.Done
+import akka.NotUsed
+import akka.actor.ActorSystem
+import akka.actor.ClassicActorSystemProvider
+import akka.actor.testkit.typed.scaladsl.LogCapturing
+import akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
+import akka.actor.testkit.typed.scaladsl.TestProbe
+import akka.projection.Projection
+import akka.projection.ProjectionId
+import akka.stream.KillSwitches
+import akka.stream.scaladsl.Sink
+import akka.stream.scaladsl.Source
+import org.scalatest.wordspec.AnyWordSpecLike
+
+class ProjectionBehaviorSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike with LogCapturing {
+
+  implicit val classicSystem: ActorSystem = testKit.system.classicSystem
```

Comment:
why is the classicSystem needed?

Reply:
leftover, I was using the stream testkit